inference-engine/thirdparty/clDNN/tests/test_cases/reorder_gpu_test.cpp

```diff
@@ -2396,6 +2399,220 @@ INSTANTIATE_TEST_SUITE_P(DISABLED_REORDER,
                         tests::generic_test::custom_param_name_functor());
 
 
+
+
+struct reorder_test_param {
+    tensor in_shape;
+    tensor out_shape;
+    tensor kernel;
+    tensor stride;
+    tensor pad;
+    data_types data_type;
+    format input_format;
+    data_types weights_type;
+    format weights_format;
+    data_types default_type;
+    format default_format;
+};
+
+template<typename T>
+class ReorderTest : public ::testing::TestWithParam<T> {
+public:
+#ifdef ENABLE_ONEDNN_FOR_GPU
+    cldnn::engine& engine = get_onednn_test_engine();
+#else
+    cldnn::engine& engine = get_test_engine();
+#endif
+    cldnn::topology topology_test;
+    cldnn::build_options bo_test;
+    static const int min_random = -200;
+    static const int max_random = 200;
+    std::vector<primitive_id> executed_prims;
+
+    void execute(T& p) {
+        auto input_prim = this->get_mem(get_input_layout(p));
+        network network_test(this->engine, this->topology_test, this->bo_test);
+        network_test.set_input_data("input", input_prim);
+
+        executed_prims = network_test.get_executed_primitive_ids();
+    }
+
+    bool check_removed(T& p, primitive_id target_id) {
+        for (auto& prim : executed_prims)
+            if (prim == target_id)
+                return false;
+
+        return true;
+    }
+
+    bool check_not_removed(T& p, primitive_id target_id) {
+        bool found = false;
+        for (auto& prim : executed_prims)
+            if (prim == target_id)
+                found = true;
+
+        return found;
+    }
```

Comment:
What about using single function for both cases, but with negation if needed?

Reply:
I wanted to have a simple name with a single usage.

Reply:
Modified